### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by Horizon1907** [ykzzyk/vision6D#19]
It seems that I haven't seen the usage documentation

**Comment by ykzzyk** [comment]
Yes, tutorial is definitely needed. Sorry, I'm being busy with work. If you need further assistance, send me an email and I will show a simple demo to you.
